ktor-network/ktor-network-quic/jvmAndNix/src/io/ktor/network/quic/streams/QUICStream.kt

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.network.quic.streams
+
+import io.ktor.network.quic.bytes.*
+import io.ktor.utils.io.bits.*
+import io.ktor.utils.io.core.*
+
+public interface QUICStream {
+    public val streamId: Long
+
+    public val output: Output
```

Comment:
may be it makes more sense to use Byte[Read/Write]Channel for I/O ?

Reply:
Hm, need to check

Reply:
I didn't find a quick way to use these interfaces here, and I need to finish my thesis by Monday, so I'll keep as it is for now, but I've added a TODO to revisit this place when I continue the work. Thank you the point though!